src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -328,7 +328,7 @@ public function get_result_messages( $queue ) {
 				_n( '%1$d new event category was created.', '%1$d new event categories were created.', $queue->activity->count( 'category', 'created' ), 'the-events-calendar' ),
 				$queue->activity->count( 'category', 'created' )
 			) .
-			' <a href="' . admin_url( 'edit.php?post_type=tribe_organizer' ) . '">' .
+			' <a href="' . admin_url( 'edit-tags.php?taxonomy=tribe_events_cat&post_type=tribe_events' ) . '">' .
```

Comment:
> Should we check if the user is running WP 4.5+ and use the `term.php` page?

Not sure if I'm misunderstanding ... but it seems to me we wouldn't want to do that, as we're not linking to the edit page for an individual term.